egret/model_library/transmission/branch.py

```diff
@@ -423,6 +456,12 @@ def declare_eq_branch_loss_ptdf_approx(model, index_set, branches, bus_p_loads,
             for gen_name in gens_by_bus[bus_name]:
                 expr -= coef * m.pg[gen_name]
 
+            # for _, phi_loss in phi_loss_from.items():
+            #     expr += coef * phi_loss
+            #
+            # for _, phi_loss in phi_loss_to.items():
+            #     expr -= coef * phi_loss
+
```

Comment:
These lines (and 440 above it) being commented out seems a bit weird to me. I don't know enough about loss calculation to know what the right math is, so if they're not needed we should eliminate them and some of the setup above them.

Reply:
Thanks for catching this -- I updated the code